Gemfile.lock

```diff
@@ -7,8 +7,8 @@ GIT
 
 GIT
   remote: https://github.com/hathitrust/hathifiles_database.git
-  revision: 9c33e648ffc171d665bd46999d00e968560b2451
-  branch: main
+  revision: 9c04f62e702187e2447c7ea645147f06c16d4651
```

Comment:
This change needs to revert to `main` for final merge after `hathifiles_database` PR is merged.